test/functional/ui/popupmenu_spec.lua

```diff
@@ -1156,10 +1156,10 @@ describe('builtin popupmenu', function()
     funcs.complete(29, {'word', 'choice', 'text', 'thing'})
     screen:expect([[
       some long prefix before the ^    |
-      {1:~                        }{n: word  }|
-      {1:~                        }{n: choice}|
-      {1:~                        }{n: text  }|
-      {1:~                        }{n: thing }|
+      {n:word           }{1:                 }|
+      {n:choice         }{1:                 }|
+      {n:text           }{1:                 }|
+      {n:thing          }{1:                 }|
```

Comment:
why did the menu position move to the left?

Reply:
https://github.com/neovim/neovim/pull/11616#discussion_r361896362